### ISSUE TITLE
CEngine missing force_retouch variable

### ISSUE THREAD

**Issue by fnky** [baso88/SC_AngelScript#19]
The [`CEngine`](https://baso88.github.io/SC_AngelScript/docs/CEngine.htm) class is missing the `force_retouch` variable. This is used to force the engine to run touch functions for non-moving entities.

**Comment by fnky** [comment]
Code to add this:

```cpp
pDocumentation->RegisterObjectProperty( "How many times to force all entities to retouch entities they are currently touching/intersecting. Should always be >= 0.", pszObjectName, 
    "float force_retouch",
    asOFFSET( globalvars_t, force_retouch ) );
```
Add after `frametime` in `trunk/dlls/Angelscript/ScriptTypes/ASEngine.cpp`

**Comment by baso88** [comment]
force_retouch variable was added and made non-const (see #20). This change will be included in the next update (5.13). Closing.
